homeassistant/components/goalzero/binary_sensor.py

```diff
@@ -5,8 +5,6 @@
 from . import YetiEntity
 from .const import BINARY_SENSOR_DICT, DATA_KEY_API, DATA_KEY_COORDINATOR, DOMAIN
 
-PARALLEL_UPDATES = 0
-
```

Comment:
Removed as this becomes none during the `_get_parallel_updates_semaphore` callback